### ISSUE TITLE
Inconsistent include guards

### ISSUE THREAD

**Issue by schlessera** [pluginkollektiv/antispam-bee#83]
The three PHP files have three different include guards.

`antispam_bee.php`:
```
/* Sicherheitsabfrage */
if ( ! class_exists( 'WP' ) ) {
	die();
}
```

`columns.class.php`:
```
/* Quit */
defined( 'ABSPATH' ) || exit;
```

`gui.class.php`:
```
/* Sicherheitsabfrage */
if ( ! class_exists('Antispam_Bee') ) {
	die();
}
```

We should decide on one and stick with it.

**Comment by websupporter** [comment]
My vote would be `ABSPATH` as the most common one.

**Comment by websupporter** [comment]
This is fixed @schlessera? If so just close it :) Thank you.

**Comment by schlessera** [comment]
Fixed in 6e3b1d7042efe99a3cd7c7534e20db05633b0e95
